variant-remapping-automation/remapping_process.nf

```diff
@@ -228,21 +230,50 @@ process cluster_studies_from_mongo {
     path ingestion_log from ingestion_log_filename
 
     output:
-    path "${params.target_assembly_accession}_clustering.properties" into clustering_props
-    path "${params.target_assembly_accession}_clustering.log" into clustering_log_filename
-    // TODO also output clustering report once it exists (EVA-2935)
+    path "${params.source_assembly_accession}_to_${params.target_assembly_accession}_clustering.properties" into clustering_props
+    path "${params.source_assembly_accession}_to_${params.target_assembly_accession}_clustering.log" into clustering_log_filename
+    path "${params.source_assembly_accession}_to_${params.target_assembly_accession}_rs_report.txt" into rs_report_filename
 
     publishDir "$params.output_dir/properties", overwrite: true, mode: "copy", pattern: "*.properties"
     publishDir "$params.output_dir/logs", overwrite: true, mode: "copy", pattern: "*.log*"
 
     """
-    cat ${params.template_properties} ${params.clustering_template_properties} > ${params.target_assembly_accession}_clustering.properties
-    echo "parameters.projectAccession=" >> ${params.target_assembly_accession}_clustering.properties
-    echo "parameters.projects=${params.studies}" >> ${params.target_assembly_accession}_clustering.properties
-    echo "spring.batch.job.names=STUDY_CLUSTERING_JOB" >> ${params.target_assembly_accession}_clustering.properties
-    echo "parameters.assemblyAccession=${params.target_assembly_accession}" >> ${params.target_assembly_accession}_clustering.properties
-    echo "parameters.remappedFrom=${params.source_assembly_accession}" >> ${params.target_assembly_accession}_clustering.properties
+    cat ${params.template_properties} ${params.clustering_template_properties} > ${params.source_assembly_accession}_to_${params.target_assembly_accession}_clustering.properties
```

Comment:
Suggest extracting "${params.source_assembly_accession}_to_${params.target_assembly_accession}" to a variable.